markdown/usage/reference_genomes.md

```diff
@@ -5,55 +5,72 @@ subtitle: How reference genomes are handled in nf-core
-Pipelines that support AWS-iGenomes can also be configured to support custom genome IDs and paths.
+As mentioned in the section above, most of the required genome assets will be defined in the [main script](https://github.com/nf-core/rnaseq/blob/e049f51f0214b2aef7624b9dd496a404a7c34d14/main.nf#L20-L28) for DSL2 nf-core pipelines. If you are unable to use the AWS iGenomes references, you can still supply reference genome parameters on the command line or via a `-params-file` in `yaml` or `json` format.
 
-To use this system, add paths to your config file using the following template:
+Using GRCh38 as an example, we can download the latest FASTA and GTF files using a simple bash script like below:
 
-```nextflow
-params {
-  genomes {
-    'YOUR-ID' {
-      fasta  = '/path/to/data/genome.fa'
-    }
-    'OTHER-GENOME' {
-      // [..]
-    }
-  }
-  // Optional - default genome. Ignored if --genome 'OTHER-GENOME' specified on command line
-  genome = 'YOUR-ID'
-}
+```bash
+#!/bin/bash
+
+VERSION=108
+wget -L ftp://ftp.ensembl.org/pub/release-$VERSION/fasta/homo_sapiens/dna/Homo_sapiens.GRCh38.dna_sm.primary_assembly.fa.gz
+wget -L ftp://ftp.ensembl.org/pub/release-$VERSION/gtf/homo_sapiens/Homo_sapiens.GRCh38.$VERSION.gtf.gz
 ```
 
-You can add as many genomes as you like as long as they have unique IDs.
-References are used with the command line option `--genome YOUR-ID`.
+Most genomics nf-core pipelines are able to start from just a FASTA and GTF file and create any downstream reference assets as part of the pipeline execution e.g. genome indices, intervals files etc. To avoid having to recreate these assets every time you run the pipeline you can use the `--save_reference` parameter that will save the indices, interval files etc in the results directory for you to move and store in a more central location for re-use with future pipeline runs. Using nf-core/rnaseq as an example [see docs](https://nf-co.re/rnaseq/output#reference-genome-files):
 
-Read the [Nextflow configuration documentation](configuration.md) for more information about custom config files.
+```bash
+nextflow run \
+    nf-core/rnaseq \
+    --input samplesheet.csv \
+    --fasta Homo_sapiens.GRCh38.dna_sm.primary_assembly.fa.gz \
+    --gtf Homo_sapiens.GRCh38.108.gtf.gz \
+    --save_reference \
+    <OTHER_PARAMETERS>
+```
+
+Any downstream reference assets will be published in the results folder. For example, if you ran the nf-core/rnaseq pipeline in the step above with default options then the STAR index will be created and stored in the `<RESULTS_DIR>/genome/index/star` folder. Once you have moved the reference files to a central location so they are persistently available you can remove the `--save_reference` parameter and now explicitly override the relevant parameters via the CLI or a `-params-file` in 'yaml' format. This will save having to re-create the genome indices and other assets over and over again which will be cost and time expensive.
```

Comment:
```suggestion
Any downstream reference assets will be published in the results folder. For example, if you ran the nf-core/rnaseq pipeline in the step above with default options then the STAR index will be created and stored in the `<RESULTS_DIR>/genome/index/star` folder. Once you have moved the reference files to a central location so they are persistently available you can remove the `--save_reference` parameter and now explicitly override the relevant parameters via the CLI or a `-params-file` in 'yaml' or `json` format. This will save having to re-create the genome indices and other assets over and over again which will be cost and time expensive.
```